src/tests/ftest/osa/offline_parallel_test.yaml

```diff
@@ -104,4 +104,4 @@ rebuild:
 checksum:
   test_with_checksum: False
 system_stop_start:
-  flags: True
\ No newline at end of file
+  flags: True
```

Comment:
Yes in theory.  This is from the "truthy" check, and I fixed some of the codebase but there are just so many files in src/test/ftest and they're all wrong that I disabled the check for this directory.  We could, either in this PR or a different one, enable this check and fix all instances however it would be time consuming and prone to conflicts so this just an easier path to getting the majority if checks enabled without expending too much effort on it.  The are similar checks for spaces after commas which are also disabled for the ftest directory, full details are in the yamllint config file.